### ISSUE TITLE
Clarify which exchange rate to use

### ISSUE THREAD

**Issue by jacobvjk** [2DegreesInvesting/PACTA_analysis#113]
https://github.com/2DegreesInvesting/PACTA_analysis/blob/1bbdc26b725f1a577c3f43fcc3a90161da0cd6b9/0_portfolio_input_check_functions.R#L462

This function calculates the holdings' `value_usd` using the `exchange_rate` from the `currencies.rda` file, whereas it should probably use the `exchange_rate_usd` from the financial data file.

We need to clarify if we can simply change the variable in use or if they are set up for different use cases.
This may require some input from AR.

**Comment by jacobvjk** [comment]
@Clare2D

**Comment by Clare2D** [comment]
Was this resolved Jacob? I know we discussed it

**Comment by jacobvjk** [comment]
We did discuss it and wanted to circle back to checking it a bit later. Which might be now?

**Comment by Clare2D** [comment]
Yep. Seems like a good time! I saw this is a PR @cjyetman is working on - so possibly we should coordinate this
